packages/fast-form-generator-react/src/utilities/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from "./form.constants.style";
+export * from "./form.utilities.style";

```

Comment:
can you move both of these files into the form folder and remove the utilities folder?